extensions/sql-migration/src/constants/queries.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can these changes be placed into the sqlUtils.ts file with other sql related changes